### ISSUE TITLE
Feature request: Add eraser to Draw Mask tool

### ISSUE THREAD

**Issue by aesterling** [VorlonCD/bi-aidetection#55]
The ability to draw a mask natively inside AI Tool is awesome, but it'd also be great if we could erase like in BI. Either by holding down a modifier key and using the brush/rectangle, or by adding a separate eraser tool. 

I often mess up when drawing a mask and would like to erase just a small portion, but instead have to click "clear" and start over. 

Please let me know if this is already possible and I missed it. Thanks!

**Comment by classObject** [comment]
@aesterling Agreed. Erase was in the original version but was removed due to technical issues. It's on my list for a future release.

**Comment by aesterling** [comment]
An addition to this request:

Add clarification text to the "draw mask" window to inform users if the mask is inclusive or exclusive. Is the mask's purpose to block out areas you don't want, or to identify areas you do want.

I understand how it works, but it can still be confusing since it's opposite of BI, so a simple reminder would be helpful.

Thoughts, @classObject? Thanks as always!

**Comment by classObject** [comment]
@aesterling That's a point of view I've never considered. I can see how comparing BI zones to masks can be confusing since in BI zones are shaded in yellow to indicte inclusion.  Masks are the inverse of zones.  I can look into ways of making that clearer. As far as the erase feature, it's still on my todo list but haven't found the time yet to implement. It will take quite a few changes to support true layers.
